### DISCUSSION TITLE
Make it easier to add titles to a playlist

### DISCUSSION THREAD

**Discussion by toby63** [FossifyOrg/General-Discussion#567] [Ideas]
For example in the titles menu there could be options to add them to a playlist.

**Comment by naveensingh** [comment]
@toby63 
What do you mean by "add titles to a playlist"? If you want to add some tracks to a playlist, there is an "Add to playlist" option for that.

**Reply by toby63** [reply]
Ah yes, my bad I'm sorry, last I checked I couldn't find such an option, but apparently you just have to mark the title, then click on the three dots and then you can add it to a playlist.
